docs/install.rst

```diff
@@ -68,17 +84,32 @@ activating your virtualenv:
 
 .. code-block:: bash
 
-  # . /opt/pyff/bin/activate
+  # source python-pyff/bin/activate
 
 Next install pyFF:
 
 .. code-block:: bash
 
-  # pip install pyFF
+  # cd pyFF
+  # LANG=en_US.UTF-8 pip install -e .
 
 This will install a bunch of dependencies and compile bindings for both lxml, pyyaml
-aswell as pyXMLSecurity. This may take some time to complete. If there are no errors and if
-you have the *pyff* binary in your **$PATH** you should be done.
+as well as pyXMLSecurity. This may take some time to complete. If there are no errors and if
+you have the *python-pyff* binary in your **$PATH** you should be done.
+
+.. code-block:: bash
+
+ # cd $HOME
+ # mkdir pyff-config
+ # cd pyff-config
+
+pyFF requires five files which you should copy into the pyff-config directory
+
+- A stylesheet in the *xsl* format
+- A metadata signer certficate
```

Comment:
there is a script in the scripts-directory for generating a signing key pair for demo purposes

Reply:
If one uses the genkey script do they have to transfer the key pair to the pyff-config directory?